src/actions/manager.tsx

```diff
@@ -10,6 +10,31 @@ import {
 import { ExcalidrawElement } from "../element/types";
 import { AppClassProperties, AppState } from "../types";
 import { MODES } from "../constants";
+import { trackEvent } from "../analytics";
+
+const trackAction = (
+  action: Action,
+  source: "ui" | "keyboard" | "api",
+  value: any,
+) => {
+  if (action.trackEvent !== false) {
+    try {
+      if (action.trackEvent === true) {
```

Comment:
when is the boolean case used ?

Reply:
not used currently. Will be used once this lands to master and I'll issue the second PR.